js/api.js

```diff
@@ -64,6 +64,9 @@ const API = {
         return mergeLayerAndMetadata();
       });
     }));  // This chain returns the layers, merged with their respective metadata.
-
   },
 }
+
+export function api() {
```

Comment:
Why not just export the API object here?

Reply:
I did not know i could export a variable, thought i had to export a function.

Reply:
`export const API = …` on the first line should work.

Reply:
marvellous and thank you :)